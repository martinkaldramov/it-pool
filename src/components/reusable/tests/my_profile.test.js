import React from 'react';
import { shallow } from 'enzyme';
import MyProfile from '../my_profile';

describe('MyProfile', () => {
  const wrap = shallow(<MyProfile />);

  it('should render successfully', () => {
    expect(wrap).toMatchSnapshot();
  });
});
