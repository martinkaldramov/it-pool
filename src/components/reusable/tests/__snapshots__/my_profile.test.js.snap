// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyProfile should render successfully 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MyProfile />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <MenuItem
          anchorOrigin={
                    Object {
                              "horizontal": "right",
                              "vertical": "top",
                            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          targetOrigin={
                    Object {
                              "horizontal": "left",
                              "vertical": "top",
                            }
          }
>
          MY PROFILE
</MenuItem>,
        <MenuItem
          anchorOrigin={
                    Object {
                              "horizontal": "right",
                              "vertical": "top",
                            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          targetOrigin={
                    Object {
                              "horizontal": "left",
                              "vertical": "top",
                            }
          }
>
          <Toggle
                    defaultToggled={true}
                    disabled={false}
                    labelPosition="left"
          />
           Receive Offers
</MenuItem>,
        <MenuItem
          anchorOrigin={
                    Object {
                              "horizontal": "right",
                              "vertical": "top",
                            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          targetOrigin={
                    Object {
                              "horizontal": "left",
                              "vertical": "top",
                            }
          }
>
          <ActionSettings />
           Settings
</MenuItem>,
        <MenuItem
          anchorOrigin={
                    Object {
                              "horizontal": "right",
                              "vertical": "top",
                            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          targetOrigin={
                    Object {
                              "horizontal": "left",
                              "vertical": "top",
                            }
          }
>
          <ImageRemoveRedEye />
           View Profile
</MenuItem>,
        <MenuItem
          anchorOrigin={
                    Object {
                              "horizontal": "right",
                              "vertical": "top",
                            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          targetOrigin={
                    Object {
                              "horizontal": "left",
                              "vertical": "top",
                            }
          }
>
          <ActionExitToApp />
           Sign Out
</MenuItem>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "anchorOrigin": Object {
            "horizontal": "right",
            "vertical": "top",
          },
          "checked": false,
          "children": "MY PROFILE",
          "desktop": false,
          "disabled": false,
          "focusState": "none",
          "insetChildren": false,
          "targetOrigin": Object {
            "horizontal": "left",
            "vertical": "top",
          },
        },
        "ref": null,
        "rendered": "MY PROFILE",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "anchorOrigin": Object {
            "horizontal": "right",
            "vertical": "top",
          },
          "checked": false,
          "children": Array [
            <Toggle
              defaultToggled={true}
              disabled={false}
              labelPosition="left"
/>,
            " Receive Offers",
          ],
          "desktop": false,
          "disabled": false,
          "focusState": "none",
          "insetChildren": false,
          "targetOrigin": Object {
            "horizontal": "left",
            "vertical": "top",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "defaultToggled": true,
              "disabled": false,
              "labelPosition": "left",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          " Receive Offers",
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "anchorOrigin": Object {
            "horizontal": "right",
            "vertical": "top",
          },
          "checked": false,
          "children": Array [
            <ActionSettings />,
            " Settings",
          ],
          "desktop": false,
          "disabled": false,
          "focusState": "none",
          "insetChildren": false,
          "targetOrigin": Object {
            "horizontal": "left",
            "vertical": "top",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          " Settings",
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "anchorOrigin": Object {
            "horizontal": "right",
            "vertical": "top",
          },
          "checked": false,
          "children": Array [
            <ImageRemoveRedEye />,
            " View Profile",
          ],
          "desktop": false,
          "disabled": false,
          "focusState": "none",
          "insetChildren": false,
          "targetOrigin": Object {
            "horizontal": "left",
            "vertical": "top",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          " View Profile",
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "anchorOrigin": Object {
            "horizontal": "right",
            "vertical": "top",
          },
          "checked": false,
          "children": Array [
            <ActionExitToApp />,
            " Sign Out",
          ],
          "desktop": false,
          "disabled": false,
          "focusState": "none",
          "insetChildren": false,
          "targetOrigin": Object {
            "horizontal": "left",
            "vertical": "top",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          " Sign Out",
        ],
        "type": [Function],
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <MenuItem
            anchorOrigin={
                        Object {
                                    "horizontal": "right",
                                    "vertical": "top",
                                  }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            targetOrigin={
                        Object {
                                    "horizontal": "left",
                                    "vertical": "top",
                                  }
            }
>
            MY PROFILE
</MenuItem>,
          <MenuItem
            anchorOrigin={
                        Object {
                                    "horizontal": "right",
                                    "vertical": "top",
                                  }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            targetOrigin={
                        Object {
                                    "horizontal": "left",
                                    "vertical": "top",
                                  }
            }
>
            <Toggle
                        defaultToggled={true}
                        disabled={false}
                        labelPosition="left"
            />
             Receive Offers
</MenuItem>,
          <MenuItem
            anchorOrigin={
                        Object {
                                    "horizontal": "right",
                                    "vertical": "top",
                                  }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            targetOrigin={
                        Object {
                                    "horizontal": "left",
                                    "vertical": "top",
                                  }
            }
>
            <ActionSettings />
             Settings
</MenuItem>,
          <MenuItem
            anchorOrigin={
                        Object {
                                    "horizontal": "right",
                                    "vertical": "top",
                                  }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            targetOrigin={
                        Object {
                                    "horizontal": "left",
                                    "vertical": "top",
                                  }
            }
>
            <ImageRemoveRedEye />
             View Profile
</MenuItem>,
          <MenuItem
            anchorOrigin={
                        Object {
                                    "horizontal": "right",
                                    "vertical": "top",
                                  }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            targetOrigin={
                        Object {
                                    "horizontal": "left",
                                    "vertical": "top",
                                  }
            }
>
            <ActionExitToApp />
             Sign Out
</MenuItem>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "anchorOrigin": Object {
              "horizontal": "right",
              "vertical": "top",
            },
            "checked": false,
            "children": "MY PROFILE",
            "desktop": false,
            "disabled": false,
            "focusState": "none",
            "insetChildren": false,
            "targetOrigin": Object {
              "horizontal": "left",
              "vertical": "top",
            },
          },
          "ref": null,
          "rendered": "MY PROFILE",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "anchorOrigin": Object {
              "horizontal": "right",
              "vertical": "top",
            },
            "checked": false,
            "children": Array [
              <Toggle
                defaultToggled={true}
                disabled={false}
                labelPosition="left"
/>,
              " Receive Offers",
            ],
            "desktop": false,
            "disabled": false,
            "focusState": "none",
            "insetChildren": false,
            "targetOrigin": Object {
              "horizontal": "left",
              "vertical": "top",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "defaultToggled": true,
                "disabled": false,
                "labelPosition": "left",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            " Receive Offers",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "anchorOrigin": Object {
              "horizontal": "right",
              "vertical": "top",
            },
            "checked": false,
            "children": Array [
              <ActionSettings />,
              " Settings",
            ],
            "desktop": false,
            "disabled": false,
            "focusState": "none",
            "insetChildren": false,
            "targetOrigin": Object {
              "horizontal": "left",
              "vertical": "top",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            " Settings",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "anchorOrigin": Object {
              "horizontal": "right",
              "vertical": "top",
            },
            "checked": false,
            "children": Array [
              <ImageRemoveRedEye />,
              " View Profile",
            ],
            "desktop": false,
            "disabled": false,
            "focusState": "none",
            "insetChildren": false,
            "targetOrigin": Object {
              "horizontal": "left",
              "vertical": "top",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            " View Profile",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "anchorOrigin": Object {
              "horizontal": "right",
              "vertical": "top",
            },
            "checked": false,
            "children": Array [
              <ActionExitToApp />,
              " Sign Out",
            ],
            "desktop": false,
            "disabled": false,
            "focusState": "none",
            "insetChildren": false,
            "targetOrigin": Object {
              "horizontal": "left",
              "vertical": "top",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            " Sign Out",
          ],
          "type": [Function],
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
